index.js

```diff
@@ -432,7 +432,24 @@ module.exports = {
 
       return routes;
     },
+    /**
+     * gets either the commonPathItemObjectResponses from settings with possibly merged responses if the flag in the openapi service is false or not specified and gets exclusively your
+     * customCommonPathItemResponses if the flag in the openapi service is true
+     * 
+     * @returns commonPathItemObjectResponses: either the commonPathItemObjectResponses from settings with possibly merged responses or exclusively your custom commonPathItemObjectResponses
+     */
+    getCommonPathItemObjectResponses(){
+      //sets the default commonPathItemObjectResponses as the commonPathItemObjectResponses from settings with possibly merged responses from openapiservice
+      let commonPathItemObjectResponses = this.settings.commonPathItemObjectResponses;
```

Comment:
please add commonPathItemObjectResponses flag with default value to settings

Reply:
okay :), i removed the identifier commonPathItemObjectResponses from getCommonPathItemObjectResponses(). And i m using the settings default instead. I also moved the useCommonPathItemObjectResponses and commonPathItemObjectResponses from settings.openapi to settings and included default values.